tectonic/dpx-dvipdfmx.c

```diff
@@ -757,10 +713,6 @@ dvipdfmx_main (int argc, char *argv[])
   pdf_load_fontmap_file("kanjix.map", FONTMAP_RMODE_APPEND);
   pdf_load_fontmap_file("ckx.map", FONTMAP_RMODE_APPEND);
 
-  /* End config file fakery. */
-
-  do_args (argc, argv, NULL, 0);
```

Comment:
Not a the moment unfortunately.`read_config_special` calls `do_args`. This function is used in `dpx-dvi.c`, it seems like it is possible to put additional command arguments in the dvi itself, which is then processed and it calls the `do_args` eventually again.

I am not sure if such feature is really used, so I took the safe choice to keep it. But we could investigate a bit more to know whether or not we could remove all of this.